### DISCUSSION TITLE
kudos!

### DISCUSSION THREAD

**Discussion by pkieltyka** [deepnoodle-ai/risor#67] [General]
I’ve been looking out for something like Risor for years — a simple, intuitive, fast and powerful scripting language to use with Go. Very impressive work

**Comment by myzie** [comment]
Thank you @pkieltyka! It's been a long road to get it to this point, and it's great hearing things like this.

Please let me know if you have any "wishlist" type features for Risor that come to mind. Any feedback like that will help guide the future direction of the project.

**Comment by alexaandru** [comment]
Looks really beautiful, both Risor and the Go code behind it: clean, nimble, well thought out. You can easily tell it was done by someone who loves their craft. Congratulations!

**Reply by myzie** [reply]
I appreciate the kind words @alexaandru 🙏 

It has been a lot of fun to work on. And it's great to hear about when it's interesting and useful for others.
